Rakefile

```diff
@@ -0,0 +1,61 @@
+desc "Release a new project version"
+task :release do
+  require 'more_core_extensions/core_ext/hash/nested'
+  require 'pathname'
+  require 'yaml'
+
+  version = ENV["RELEASE_VERSION"]
+
+  if version.nil? || version.empty?
+    STDERR.puts "ERROR: You must set the env var RELEASE_VERSION to the proper value."
+    exit 1
+  end
+
+  if `git rev-parse --abbrev-ref HEAD`.chomp == "master"
+    STDERR.puts "ERROR: You cannot cut a release from the master branch."
+    exit 1
+  end
+
+  branch, minor_patch, milestone = version.split("-")
+  major = (branch[0].ord - 96).to_s     # ivanchuk = 9, jansa = 10
```

Comment:
this approach will break in the release after Z or 27, but we can worry about that when we get there.